library/src/main/java/pl/netigen/premium/PremiumItemNetigenApi.java

```diff
@@ -0,0 +1,37 @@
+package pl.netigen.premium;
+
+public class PremiumItemNetigenApi {
```

Comment:
It's already in NetigenApi it doesn't have to be named after NetigenApi. PremiumItem could be enough